scipy/stats/tests/test_stats.py

```diff
@@ -7663,6 +7663,20 @@ def test_error_code(self):
             v_values = rng.random(size=(2, 2))
             _ = stats.wasserstein_distance(u_values, v_values)
 
+    def test_flatten(self):
+        rng = np.random.default_rng(456347456756786)
+        num_tests = 100
```

Comment:
Rather than manually looping, consider using `pytest.parametrize`. Also, 100 tests is probably excessive. We're confident that the two backends agree now, and it is likely that a regression would be caught with one or a few tests.

```suggestion
    @pytest.mark.parametrize('seed', [45634745675678, 67864563474567, 56756786456347]
    def test_flatten(self):
        rng = np.random.default_rng(seed)
```